db/migrate/20230621191706_add_role_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUser < ActiveRecord::Migration[7.0]
```

Comment:
- [ ] Kindly fix this error that  is in your migrations
<img width="775" alt="image" src="https://github.com/DavidEspinoG/rails_blog_microverse/assets/65068771/a80eea52-e1ad-4277-b0c1-9e047fb60504">